lib/trader.js

```diff
@@ -231,6 +376,18 @@ function isNetworkError (err) {
   }
 }
 
+function isNetworkError (err) {
+  return !!err.networkError
+}
+
+/*
+ * TODO: Remove once the old poller is removed -- replace `this.isNetworkError`
+ * with just `isNetworkError` everywhere!
+ */
```

Comment:
Can you add the deprecation comment on related places? Check how we did with T&C